joomla/joomla-cms
# Fix button focus boder cut off

Pull Request for Issue #37706.

### Summary of Changes
I added a 0.25 rem distance to the left of the toolbar so that the focus border is not cut off.

### Testing Instructions
Apply patch and in Cassiopeia Frontend in the Media Modal window the Upload Button should have a full focus border

### Actual result BEFORE applying this Pull Request
<img width="372" alt="grafik" src="https://user-images.githubusercontent.com/828371/166141655-48765f6c-6205-4359-911b-2345b45e7bad.png">

### Expected result AFTER applying this Pull Request
<img width="510" alt="grafik" src="https://user-images.githubusercontent.com/828371/166141649-85fdddfc-2cc3-44b5-92db-985d289c2280.png">

### Documentation Changes Required
No
